src/components/Editor.js

```diff
@@ -6,28 +6,33 @@ import {styles} from './styles';
 import Toolbar from './Toolbar';
 import CodeWindow from './CodeWindow';
 import Spinner from './Spinner';
+import ErrorBox from './ErrorBox';
 
 class Editor extends React.Component {
   render() {
     return (
       <div>
-        <Toolbar logo="https://code.pyret.org/img/pyret-logo.png" />
-        {this.props.isLoadingRuntime &&
-         <Spinner style={styles.spinners.window} />
-        }
-        {this.props.hasLoadedRuntime &&
-         <div>
-           <SplitPane defaultSize="50%" split="vertical">
-             <div><CodeWindow/></div>
-             <div><p>{this.props.result}</p></div>
-           </SplitPane>
-         </div>
-        }
+          <Toolbar logo="https:code.pyret.org/img/pyret-logo.png" />
+          {this.props.isLoadingRuntime &&
+           <Spinner style={styles.spinners.window} />
+          }
+           {this.props.hasLoadedRuntime &&
+            <div>
+                <SplitPane defaultSize="50%" split="vertical">
+                    <div><CodeWindow/></div>
+                    <div>
+                        <ErrorBox/>
+                        <p>{this.props.result}</p>
+                    </div>
+                </SplitPane>
+            </div>
+           }
```

Comment:
I think you need to fix your web-mode config to use 2 spaces for markup:

`M-x customize-variable RET web-mode-markup-indent-offset`